packages/sdk/index.mjs

```diff
@@ -0,0 +1 @@
+export * from './dist/sdk.cjs.js';
```

Comment:
@nickrum Same problem here as with the format-title one?

```suggestion
import DirectusSDK from './dist/sdk.cjs.js';
export default DirectusSDK;
```

Reply:
The SDK only has named exports, so this should be fine (I actually copied over this file from the sdk to format-title, that's how the issue with the format-title one snuck in).